libraries/AP_HAL_ChibiOS/RCOutput.cpp

```diff
@@ -1612,4 +1597,67 @@ uint32_t RCOutput::protocol_bitrate(const enum output_mode mode) const
         return 1;
     }
 }
+
+/*
+  setup neopixel (WS2812B) output for a given channel number, with
+  the given max number of LEDs in the chain.
+*/
+bool RCOutput::set_neopixel_num_LEDs(const uint16_t chan, uint8_t num_leds)
+{
+    uint8_t i;
+    pwm_group *grp = find_chan(chan, i);
+    if (!grp) {
+        return false;
+    }
+    if (grp->neopixel_nleds == num_leds && grp->current_mode == MODE_NEOPIXEL) {
```

Comment:
does this need an (... && num_leds>0) check?

Reply:
it should all work with zero, will end up sending just the pad bits